cmd/system-probe/config/adjust_usm.go

```diff
@@ -71,6 +71,15 @@ func adjustUSM(cfg model.Config) {
 	applyDefault(cfg, smNS("max_postgres_stats_buffered"), 100000)
 	applyDefault(cfg, smNS("max_redis_stats_buffered"), 100000)
 
+	// kernel_buffer_pages determines the number of pages allocated *per CPU*
+	// for buffering kernel data, whether using a perf buffer or a ring buffer.
+	applyDefault(cfg, smNS("kernel_buffer_pages"), 16)
```

Comment:
why using apply default when the value is new?

Reply:
It had default values before, so I need to keep backward compatilibity

Reply:
Also, using `applyDefault` is preferable, as it allows you to distinguish between a scenario where a customer hasn't set this value and one where they have explicitly set the default value.